.github/workflows/validate-pr.yml

```diff
@@ -40,31 +40,33 @@ jobs:
         run: |
           orgflow env:list --output=json | jq . -c | ( read output; echo "::set-output name=environments::$output"; )
 
-      # Find the environment the source branch belongs to:
+      # Find the environment that the source branch belongs to:
       - name: Find source environment
         id: find-source-environment
         run: |
-          echo '${{ steps.env-list.outputs.environments }}' |
-          jq '[.[] | select(.git.branch == "${{ github.head_ref }}") | .name] | select(. | length > 0)[0]' -r | 
-          ( read output; echo "::set-output name=environment-name::$output"; )
-
-      # ... and same for the target branch:
+          set -e          
+          output=`echo '${{ steps.env-list.outputs.environments }}' | jq '[.[] | select(.git.branch == "${{ github.head_ref }}") | .name] | select(. | length > 0)[0]' -r` 
+          echo "::set-output name=environment-name::$output";
+          
+      # ... and then again for the target branch:
       - name: Find target environment
         id: find-target-environment
         run: |
-          echo '${{ steps.env-list.outputs.environments }}' |
-          jq '[.[] | select(.git.branch == "${{ github.base_ref }}") | .name] | select(. | length > 0)[0]' -r | 
-          ( read output; echo "::set-output name=environment-name::$output"; )
+          set -e          
+          output=`echo '${{ steps.env-list.outputs.environments }}' | jq '[.[] | select(.git.branch == "${{ github.base_ref }}") | .name] | select(. | length > 0)[0]' -r` 
+          echo "::set-output name=environment-name::$output";
+
+      # Print a message if at least one branch does not correspond to an environment:
+      - name: End with success if one of the branches does not correspond an OrgFlow environment

```

Comment:
```suggestion
      - name: Ensure both branches are OrgFlow environments
```

(Just to keep step names roughly the same length and verbosity.)

Reply:
Yeah the name of this step is awkward because:
- it doesn't (and can't) end the job
- it's just a nicety to make it more explicit why subsequent steps are skipped
- this step is skipped if the branches are OrgFlow branches, which makes wording along the lines of 'ensure' or 'check' awkward

In the end I went with 'End with success' because it produces much clearer output in the job's log. i.e. you can see why subsequent steps are skipped, and this step is only ever executed in this case.

Reply:
eg OrgFlow branches:

![image](https://user-images.githubusercontent.com/2872984/193807131-005e581c-5b98-4ea1-b16c-792c7e37ea21.png)

and non-OrgFlow branches

![image](https://user-images.githubusercontent.com/2872984/193807205-8f66fd23-03fd-4f74-8005-1855a0b66141.png)